meetings/api/views.py

```diff
@@ -7,17 +7,43 @@
 from rest_framework.response import Response
 from django.contrib.auth import authenticate, login
 from rest_framework import status
+from allauth.socialaccount.models import SocialToken
+from allauth.socialaccount.models import SocialAccount
+from .apps import OsfOauth2AdapterConfig
 
 
 class checkLoggedIn(APIView):
+
     def get(self, request, format=None):
         if request.user.is_authenticated():
             return Response('true')
         else:
             return Response('false')
 
 
+class viewCurrentUser(APIView):
+    base_url = '{}oauth2/{}'.format(
+        OsfOauth2AdapterConfig.osf_accounts_url, '{}')
+    access_token_url = base_url.format('token')
+    authorize_url = base_url.format('authorize')
+    profile_url = '{}v2/users/me/'.format(OsfOauth2AdapterConfig.osf_api_url)
+
+    def get(self, request, **kwargs):
+        print "Got Get"
+        if request.user.is_authenticated():
+            curUser = request.user.username
+            account = SocialAccount.objects.get(uid=curUser)
+            token = SocialToken.objects.get(account=account)
+            extra_data = requests.get(self.profile_url, headers={
```

Comment:
https://github.com/CenterForOpenScience/osf-meetings/pull/27/commits/7d920a0a28b272bdf127311c1cb5d7011ce792c2

Got it with this one, accidentally deleted the import somewhere in here.